src/consensus/pbft/libbyz/View_info.cpp

```diff
@@ -361,11 +361,10 @@ bool View_info::add(std::unique_ptr<View_change> vc)
   if (n.view() == vcv)
   {
     // There is a new-view message corresponding to "vc"
-    bool verified = vc->verify();
-    stored = n.can_add(vc.get(), verified);
+    stored = n.can_add(vc.get(), true);
     if (stored)
     {
-      n.add(std::move(vc), verified);
+      n.add(std::move(vc), true);
```

Comment:
Since we are not verifying anymore why not remove the `verified` function argument from `add` and `can_add`